app/views/devise/sessions/new.html.slim

```diff
@@ -1,12 +1,12 @@
 = form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-  .container align='center' 
+  .container align='center'
     .panel.panel-info
       .panel-heading
         .panel-title Sign In
-        div style=('float:right; font-size: 80%; position: relative; top:-10px') 
-      .panel-body style='padding-top:30px' 
-        #login-alert.alert.alert-danger.col-sm-12 style='display:none' 
-        form#loginform.form-horizontal role='form' 
+        div style=('float:right; font-size: 80%; position: relative; top:-10px')
```

Comment:
It's very bad practice to keep `css` styles in the `HTML`. You need to create a new PR to move all styles from `HTML` to appropriate `css` files